libft/tests/strings/ft_split_skip_delim_test.c

```diff
@@ -0,0 +1,85 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_split_skip_delim_test.c                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/23 16:24:51 by taretiuk          #+#    #+#             */
+/*   Updated: 2024/10/23 18:03:26 by taretiuk         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../sources/strings/strings.h"
+#include "tests.h"
+
+#define NUM_TEST_CASES 2
+#define MAX_ARGS 9
+
+
+typedef struct s_string
+{
+	char	*str;
+}	t_string;
+
+typedef struct s_string_array
+{
+	t_string	*strs;
+	int			error;
+	size_t		count;
+}	t_strings;
+
+static t_strings	create_string_array(void)
+{
+	t_strings	str_array;
+	str_array.count = 1;
+	str_array.error = 0;
+	str_array.strs = ft_calloc_if(sizeof(t_string) * str_array.count, 1);
+	if (str_array.strs == NULL)
+	{
+		fprintf(stderr, "Memory allocation failed for str_array.strs\n");
+		str_array.error = 1;
+		str_array.strs = NULL;
+		return (str_array);
+	}
+	for (int i = 0; i < (int)str_array.count; i++)
+	{
+		str_array.strs[i].str = NULL;
+	}
+	str_array.strs[0].str = strdup("echo hello world");
```

Comment:
even if we don;t have to follow the norme in tests, please keep indents proper :)